meetings/read/get_example_response.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
please let me know if this naming convention is improper, i felt some clarity would be better than having 2 files named `example_response.json`